src/Mobs/Path.cpp

```diff
@@ -6,6 +6,7 @@
 #include "Path.h"
 #include "../Chunk.h"
 
+#define JUMP_G_COST 20
```

Comment:
Going up costs more then going straight, so mobs now avoid minor obstacles (3.g. 1 block in the way) rather than jumping on them.